modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/AppContext.es.js

```diff
@@ -179,26 +180,32 @@ const setDataDefinitionFields = (
 	dataDefinition,
 	dataLayout
 ) => {
+	function requiredField(fieldName) {
+		const field = getDataDefinitionField(dataDefinition, fieldName);
+
+		return field?.required ?? false;
```

Comment:
We should keep !! as boolean checker, or use ?? and then the default value?

Resumed

```
!!field?.required
```

instead of

```
field?.required ?? false;
```

We used this **!!** approach before, what you think @interaminense ?